### ISSUE TITLE
[Proposal] provide out-of-box grafana dashboards

### ISSUE THREAD

**Issue by ptyin** [apache/incubator-seata#6007]
## Why you need it?

Though we already have metrics service and prometheus exporter, there is still one more work to do in terms of observability, i.e. a visualizer. Nowadays, users have to maintain a self-made grafana panel to observe Seata metrics.

## How it could be?

I propose to implement an out-of-box grafana panel for users to quickly get hands on. And we can integrate the grafana panel to Seata console, to make Seata console a one-stop control plane for Seata transaction.

The grafana panel should be like something in this blog [post](https://seata.io/zh-cn/blog/seata-observable-practice).
![image](https://github.com/seata/seata/assets/32592585/051ee183-ecf6-4ef4-9f05-9fe93c87255b)

## Other related information

How do you guys think?

**Comment by leizhiyuan** [comment]
In my opinion

First, we need to sort out all the metrics, similar to https://pulsar.apache.org/docs/3.1.x/reference-metrics/.

Then, provide a Grafana JSON configuration template that can be submitted to https://grafana.com/grafana/dashboards/ for maintenance.

**Comment by funky-eyes** [comment]
Agree with @leizhiyuan  proposal

**Comment by ptyin** [comment]
> In my opinion
> 
> First, we need to sort out all the metrics, similar to https://pulsar.apache.org/docs/3.1.x/reference-metrics/.
> 
> Then, provide a Grafana JSON configuration template that can be submitted to https://grafana.com/grafana/dashboards/ for maintenance.

Agreed. Also, the first step has already been done, since all supported metrics can be found in this [doc](https://github.com/seata/seata/blob/2.x/metrics/README.md).

**Comment by funky-eyes** [comment]
https://github.com/seata/seata/pull/4878   The client side has some future metrics

**Comment by ptyin** [comment]
A followup here. A demo grafana panel that integrates current metrics:

<img width="1289" alt="image" src="https://github.com/seata/seata/assets/32592585/e2ce0153-771b-4872-a0df-0770308bc261">

**Comment by lostpaddle** [comment]
> A followup here. A demo grafana panel that integrates current metrics:
> 
> <img alt="image" width="1289" src="https://private-user-images.githubusercontent.com/32592585/282246047-e2ce0153-771b-4872-a0df-0770308bc261.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1MDY0MzMsIm5iZiI6MTcyMjUwNjEzMywicGF0aCI6Ii8zMjU5MjU4NS8yODIyNDYwNDctZTJjZTAxNTMtNzcxYi00ODcyLWEwZGYtMDc3MDMwOGJjMjYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAxVDA5NTUzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWExYjFhNzZlZTIwNjdhMWMwMWE0ZjZkYWRmMTVmZjZiMDdiM2E3YjBiMDc4OTFhNzU1MTNlODUzZDcyZjQ1M2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1Svuc3E_pHnt9cjLuMGKY2Ym84SHb0DJ84P4Umm2_D8">

could you share this grafana json config please?
